src/symbolize/libbacktrace.rs

```diff
@@ -400,8 +400,17 @@ unsafe fn init_state() -> *mut bt::backtrace_state {
                     Ok(&buf[..(len + 1) as usize])
                 }
             }
-
-
+        } else if #[cfg(target_os = "vxworks")] {
+            unsafe fn load_filename() -> *const libc::c_char {
+                use libc;
+                use core::mem;
+                let mut rtp_desc : libc::RTP_DESC = mem::zeroed();
```

Comment:
Ah the beauty of type inference! If I'm understanding this right, mem::zeroed() automatically turns into mem::zeroed<libc::RTP_DESC>(), due to the type on the lhs. Cool.